src/System.IO.Compression.Brotli/System/IO/Compression/BrotliStream.cs

```diff
@@ -317,17 +308,18 @@ public override void Write(byte[] buffer, int offset, int count)
                 copyLen = bytesRemain > _bufferSize ? _bufferSize : bytesRemain;
                 byte[] bufferInput = new byte[copyLen];
                 Array.Copy(buffer, currentOffset, bufferInput, 0, copyLen);
-                bytesRemain -= copyLen;
-                currentOffset += copyLen;
-                _availableInput = copyLen;
-                _availableOutput = _bufferSize;
                 TransformationStatus transformationResult = TransformationStatus.DestinationTooSmall;
-                while (transformationResult == TransformationStatus.DestinationTooSmall)
+                /*Span<byte> bufferInput = new Span<byte>(buffer);
+                bufferInput.Slice(0, copyLen);*/
```

Comment:
It was a question to @shiftylogic Can I use smth like this instead of allocate and copy array?